backend/danswer/search/pipeline.py

```diff
@@ -323,6 +399,20 @@ def relevant_section_indices(self) -> list[int]:
         )
         return self._relevant_section_indices
 
+    @property
+    def relevance_summaries(self) -> dict[str, RelevanceChunk]:
```

Comment:
can check if DISABLE_AGENTIC_SEARCH if set here and raise an exception if it's disabled